packages/aft/README.md

```diff
@@ -20,34 +20,41 @@ A CLI tool for managing the Amplify Flutter repository.
 - `run`: Run a script defined in `aft.yaml`
 - `version-bump`: Bumps version using git history
 
-## Setup
+A full list of available commands and options can be found by running `aft --help`.
+
+## Development
 
-To run some commands, `libgit2` is required and can be installed with the following commands:
+When developing `aft`, some commands will require the `libgit2` library for interacting with the repo's commit history. This library can be installed with the following commands:
 
 ```sh
+# macOS
 $ brew install libgit2
 ```
 
+> Note: If you encounter a dependency error after install for libgit2dart, run `git submodule update --init`.
```

Comment:
```suggestion
> Note: If you encounter a dependency error during install for libgit2dart, run `git submodule update --init`.
```

Reply:
Also, I would move this to after the line beginning `To activate and run`, because that's when this type of error would be encountered (when `pub get` or `pub activate` is run)